bokehjs/src/lib/models/widgets/div.ts

```diff
@@ -1,14 +1,36 @@
 import {Markup, MarkupView} from "./markup"
 import * as p from "core/properties"
+import {BundleProvider, MathJaxProvider} from "models/text/providers"
+import {find_math_parts} from "models/text/utils"
+import {TeX} from "models"
 
+const default_provider: MathJaxProvider = new BundleProvider()
```

Comment:
The default provider must be shared between all models that use MathJax. Perhaps you may want to move `default_provider` to providers.